act/event.json

```diff
@@ -0,0 +1,10 @@
+{
+  "action": "opened",
+  "repository": {
+    "owner": {
+      "login": "cds-snc"
+    },
+    "name": "cds-ai-codereviewer"
+  },
+  "number": 3
+}
```

Comment:
This contains the github `opened` action event for [act](https://github.com/nektos/act/).